libs/pages/clusters/src/lib/feature/page-clusters-create-feature/step-features-feature/step-features-feature.tsx

```diff
@@ -50,22 +51,30 @@ export function StepFeaturesFeature() {
   })
 
   const onSubmit = methods.handleSubmit((data) => {
-    if (generalData?.cloud_provider === 'AWS' || generalData?.cloud_provider === 'GCP') {
-      if (data.vpc_mode === 'DEFAULT') {
+    if (
+      generalData?.cloud_provider === 'AWS' ||
+      generalData?.cloud_provider === 'GCP' ||
+      (generalData?.cloud_provider as string) === 'SCW'
```

Comment:
The cast isn't necessary ?


```suggestion
      generalData?.cloud_provider === 'SCW'
```